plugins/synapticsmst/synapticsmst-device.h

```diff
@@ -78,8 +78,17 @@ const gchar 	*synapticsmst_device_get_chip_id 		(SynapticsMSTDevice *device);
 const gchar 	*synapticsmst_device_get_aux_node		(SynapticsMSTDevice *device);
 guint16 	 synapticsmst_device_get_rad 			(SynapticsMSTDevice *device);
 guint8 		 synapticsmst_device_get_layer 			(SynapticsMSTDevice *device);
-gboolean
-synapticsmst_device_get_cascade					(SynapticsMSTDevice *device);
+gboolean synapticsmst_device_get_cascade					(SynapticsMSTDevice *device);
+//Add for Panamera
```

Comment:
This comment isn't necessary.
Please keep the same style for all these function indentations as already in this (lots of whitespace is off)